rfc/github-cdevents.md

```diff
@@ -0,0 +1,58 @@
+
+### Configure webhook
+Everything concerning the webhook's configuration is explained in the Hookdeck [quickstart](https://hookdeck.com/webhooks/platforms/tutorial-github-webhooks) tutorial. 
+ ```config
+[remote "github-events"]
+  Dashboard = https://console.hookdeck.com
+  Source: https://hkdk.events/75yvzpqic09vvw
+  Connections: github -> github_to_cli-github forwarding to /github-webhooks-endpoint
+  Secret: 1234ABCD
+```
+for additional information you can refer to the [Hookdeck](https://hookdeck.com/webhooks/platforms/tutorial-github-webhooks) configuration page and the 
+[github-webhook](https://docs.github.com/en/webhooks/using-webhooks/creating-webhooks) page.
+
+### CDEvent github-webhook-tranlator 
+
+A new REST endpoint `/github-events` will be implemented to receive all github-events and translate them to CDEvents.
+The below list of [CDEvents-Source Code Control Events](https://github.com/cdevents/spec/blob/v0.3.0/source-code-version-control.md) will be mapped with corresponding Github-Webhook event types to translate.  
+Since this effort is to process [webhook events](https://docs.github.com/en/webhooks/webhook-events-and-payloads), we consider processing the webhook-events 
+related to branch, repository, deployment and artifact. The [CDEvent-Spec](https://github.com/cdevents/spec/blob/v0.3.0/spec.md) does not have events for tags or release yet.
+
+
+| CDEvent Type                     | Webhook/Github Event Type   | Comments                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+|:---------------------------------|:----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| dev.cdevents.repository.forked   | fork                        | A user forks a repository. (should be defined in the spec)                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
We could use the `repository.created` event here. There is no `repository.forked` event in CDEvents and I'm not sure we should add one as it's a very GitHub specific thing. 
We could have add a reference to a source or template repo in the `repository.created` event if that helps.